### PR TITLE
So/new home page

### LINKED ISSUE #17: Create page: Homepage
image

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?
- Created homepage with responsive styles
- Applied @mantagen 's PR comment regarding site wide font 
- **NB.** black background applied until a fix is found to display the background image, if we can go through that together that would be ace

### Which issue(s) does this PR fix?

Fixes #17

### Screenshots (if there are design changes)
Screenshot 2020-07-06 at 16 21 26
Screenshot 2020-07-06 at 16 22 27

### How to test
`yarn start` and click and drag using dev tools responsive view

### CODE REVIEW

File: src/components/Home.tsx [hunk 1]
```diff
@@ -1,41 +1,125 @@
...
+        left={30}
+        bottom={30}
+        border={1}
+        borderColor="athensGray"
+        borderStyle="solid"
+      >
+        BUY
+      </BuyButton>
+      <Logo
+        width={[100, 120, 200, 280]}
```
Comment (mantagen) [inline src/components/Home.tsx:115]: (See image above) there is a little bit extra space at the bottom of this container.
If it `display: flex` and that space will go away

File: src/App.tsx [hunk 2]
```diff
@@ -1,19 +1,76 @@
...
   return (
     <Router>
       <ThemeProvider theme={theme}>
         <div className="App">
           <Suspense fallback={<div>Loading</div>}>
-            <NavMenu />
+            <Container
+              position="absolute"
+              right={[75, 75, 75, 100, 100]}
+              top={28}
```
Comment (mantagen) [inline src/App.tsx:54]: Feel like right and top should be `30` here to be consistent with the content in the other four corners?
image

Comment (sssoja) [reply src/App.tsx:54]: This was to leave space for the menu button once merged into the `develop` branch! I feel you on the `top` prop though.

File: src/components/Home.tsx (hunk 1 above)
Comment (sssoja) [reply src/components/Home.tsx:115]: Eagle-eyed @mantagen!

Comment (sssoja) [reply src/components/Home.tsx:115]: + Applied the same to the oxymore logo in the top left

**Pull request merged**